modules/indexer/conversations/bleve/bleve.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
```

Comment:
I don't know why we need indexer for commit comments. when and where will these indexers be used?